composer.json

```diff
@@ -43,6 +43,7 @@
     "autoload": {
         "psr-4": {
             "PHPLRPM\\": "src/",
+            "PHPLRPM\\Cluster\\": "src/cluster/",
```

Comment:
It's better to have capitalized folder name - `Cluster`. Then it will be resolved based on the previous path, and you don't need this line.

Reply:
That's a good tip. Might be even better to move this whole clustering functionality to a separate package -- it's just another `ConfigurationSource`, loosely coopled to lrpm through that interface.

Reply:
Moved code to separate repo:

https://github.com/vrza/php-lrpm-cluster